src/journald.c

```diff
@@ -229,8 +224,7 @@ var = NULL
     vars(safe_free);
 #undef safe_free
 
-    if (event)
-        g_object_unref(event);
+    eventd_event_unref(event);
```

Comment:
Thanks for that; GObject really knows how to make an API of a thousand papercuts :) .